vdyp-sindex/sindxdll.java

```diff
@@ -0,0 +1,3947 @@
+public class sindxdll{

```

Comment:
Java classes should really follow the upper camel case convention.  This should also be part of a package and a Maven module as part of the overall build.

Reply:
I'm not terribly familiar with Maven. Have I set this up correctly?

Reply:
You've got the module's POM correct, you just need to add an entry int he parent POM itself referencing the module. Like this: https://github.com/bcgov/VDYP/blob/main/pom.xml#L25